Marlin/Configuration_adv.h

```diff
@@ -2275,6 +2275,166 @@
   #define E6_HYBRID_THRESHOLD     30
   #define E7_HYBRID_THRESHOLD     30
 
+  /**
+   * CoolStep. Currently supported for TMC2130, TMC2209, TMC5130 and TMC5160 only.
+   * This mode allows for cooler steppers and energy savings.
+   * The driver will switch to coolStep when stepper speed is over COOLSTEP_THRESHOLD mm/s.  
+   * 
+   * If SG_RESULT goes below COOLSTEP_LOWER_LOAD_THRESHOLD * 32 stepper current will be increased.
+   * Set to 0 to dable CoolStep.
```

Comment:
Should we set the defaults so CoolStep is disabled?

Reply:
I'm not sure. I guess they do need to be tuned. The current behavior is to have them disabled, so I guess it makes sense to remain consistent.

Reply:
Hmm, actually I think it might be good to leave these on and get them tuned in bugfix. Maybe they will be disabled in the release, but we'll see….

Reply:
Yes they have to be tuned to specific stepper back EMF by checking SG_RESULT at typical speeds.
To disable it by default we need COOLSTEP_LOWER_LOAD_THRESHOLD  to be 0.